### ISSUE TITLE
-Dcmake_prefix_path does not work for CMake subprojects

### ISSUE THREAD

**Issue by H5117** [mesonbuild/meson#7249]
This works for both CMake dependencies and CMake subprojects:
```
$ export CMAKE_PREFIX_PATH=<somedir>
$ meson <builddir>
```
And this works for CMake dependencies only:
```
meson -Dcmake_prefix_path=<somedir> <builddir>
```
I think it should be mentioned in the docs to avoid confusion.

**Comment by mensinda** [comment]
Nope, this is an oversight on our part. `cmake_prefix_path` should also affect CMake subprojects.

**Comment by mensinda** [comment]
Should be fixed with #7254.
